cluster/cluster_test.go

```diff
@@ -242,3 +242,52 @@ func TestClient_DetectLinkerd(t *testing.T) {
 		})
 	}
 }
+
+func TestClient_DetectTraefik(t *testing.T) {
+	data := []struct {
+		name           string
+		clientset      kubernetes.Interface
+		expectedResult bool
+	}{
+		{
+			name:           "No Traefik",
+			clientset:      fake.NewSimpleClientset(),
+			expectedResult: false,
+		},
+		{
```

Comment:
Might be good to add a test here for where Traefik is installed but not in the `traefik-v2` namespace to document the expected behaviour @Tarick